### DISCUSSION TITLE
MW4 Ver 3.0, Postage Stamp, titled ver 2.2.7

### DISCUSSION THREAD

**Discussion by JDAstroPhoto** [mworion/MountWizzard4#258] [Q&A] [answered]
I used the startup.pyz file under the "3.0" directory to install MW4 3.0.
Results:
1. The main menus is postage stamp, and it will not expand.  Is there a file I can change to make the menu bigger.  At the current size it is unusable.  
2. It looks like the main menu still says version 2.2.7, instead of 3.0.  Is that the same for everyone else?
Thanks Michel for you work.  I really enjoy using MW4

**Comment by mworion** [comment, marked as answer]
Hi,
thanks for the feedback. Did you use the scripts 2.0 to run MW4 and it will show up in correct size?
If so this would explain what to adjust. But there are setting to adjust the size. I uploaded the next beta of the script, which should reflect this (same location). Just copy it over and start again.
The settings to manually adjust the HDPI scaling in Windows you could see with 
```
python3 startup.pyz --help
```
These are
```
python3 startup.pyz --scale 2  # for example to double the size of the appearance 
python3 startup.pyz --dpi 48  # for example to double the size of the font
```

To your second question. I'm currently testing v3 and a beta will be available soon. 
Michel

**Reply by JDAstroPhoto** [reply]
Thank-you for the response, I was mistaken, I thought 3.0 was already out.  I use the Windows scripts for 2.2.7 and the sizing came out to 3/4 screen, which is sufficient.  
This was the first time I used the startup.pyz script.  I look forward to 3.0.

**Reply by mworion** [reply]
Thanks for trying. It is still not perfect, I will add the support of aarch64 (Raspi 4) as well for the script.

**Comment by Gomezrbz** [comment]
hmmmm they caught me?
